packages/manager/apps/telecom/src/components/telecom/telephony/sidebar/telephony-sidebar.service.js

```diff
@@ -24,92 +23,78 @@ angular.module('managerApp').service('TelephonySidebar', function TelephonySideb
+            .localeCompare(second.getDisplayedName()));
+
+        sortedBillingAccounts.forEach((billingAccount) => {
+          // create subsection object
+          const billingAccountSubSection = SidebarMenu.addMenuItem({
+            id: billingAccount.billingAccount,
+            title: billingAccount.getDisplayedName(),
+            state: 'telecom.telephony',
+            stateParams: {
+              billingAccount: billingAccount.billingAccount,
+            },
+            allowSubItems: billingAccount.services.length > 0,
+          }, self.mainSectionItem);
+
+          /* ----------  Numbers (alias) display  ---------- */
+
+          // first sort numbers of the billingAccount
+          const sortedAlias = tucVoipService
+            .constructor.sortServicesByDisplayedName(billingAccount.getAlias());
+
+          // add number subsections to billingAccount subsection
+          addServiceMenuItems(sortedAlias, {
+            state: 'telecom.telephony.alias',
+            prefix: $translate.instant('telecom_sidebar_section_telephony_number'),
+          }, billingAccountSubSection);
+
+          /* ----------  Lines display  ---------- */
+
+          // first sort lines of the billingAccount
+          const sortedLines = tucVoipService
+            .constructor.sortServicesByDisplayedName(billingAccount.getLines());
+
+          // display lines except plugAndFax and fax
+          const sortedSipLines = filter(sortedLines, line => ['plugAndFax', 'fax', 'voicefax'].indexOf(line.featureType) === -1);
+
+          // add line subsections to billingAccount subsection
+          const sipTrunkPrefix = $translate.instant('telecom_sidebar_section_telephony_trunk');
+          const sipPrefix = $translate.instant('telecom_sidebar_section_telephony_line');
+
+          addServiceMenuItems(sortedSipLines, {
+            state: 'telecom.telephony.line',
+            prefix(lineService) {
+              return lineService.isSipTrunk() ? sipTrunkPrefix : sipPrefix;
+            },
+          }, billingAccountSubSection);
+
+          // second get plugAndFax
+          const sortedPlugAndFaxLines = tucVoipService.constructor
+            .filterPlugAndFaxServices(sortedLines);
+
+          // add plugAndFax subsections to billingAccount subsection
+          addServiceMenuItems(sortedPlugAndFaxLines, {
+            state: 'telecom.telephony.line',
+            prefix: $translate.instant('telecom_sidebar_section_telephony_plug_fax'),
+          }, billingAccountSubSection);
+
+          // then get fax
+          const sortedFaxLines = tucVoipService.constructor.filterFaxServices(sortedLines);
+
+          // add fax subsections to billingAccount subsection
+          addServiceMenuItems(sortedFaxLines, {
+            state: 'telecom.telephony.fax',
+            prefix: $translate.instant('telecom_sidebar_section_telephony_fax'),
+          }, billingAccountSubSection);
+        });
+      });
```

Comment:
Indentation changes

Reply:
Maybe two commits would've been indicated in that case (to improve readability between the fix and the style changes)

Reply:
Indentation changes because I remove https://github.com/ovh-ux/manager/pull/1485/files#diff-e3dadd8546046ad7b52489f7ec33587dL30 ;)